src/build/prepareElectronApp.ts

```diff
@@ -92,33 +94,37 @@ async function maybeCopyScripts(srcs: string[], dest: string): Promise<void> {
       );
     }
 
-    let destFileName: string;
-    if (path.extname(src) === '.js') {
-      destFileName = 'inject.js';
-    } else if (path.extname(src) === '.css') {
-      destFileName = 'inject.css';
-    } else {
-      return;
+    if (supportedInjectionExtensions.indexOf(path.extname(src)) < 0) {
+      console.log(`Skipping unsupported injection file: ${src}`);
+      continue;
     }
 
+    const postFixHash = generatePostFixHash(src);
+    const destFileName = `inject-${postFixHash}.${path.extname(src)}`;
     const destPath = path.join(dest, 'inject', destFileName);
     log.debug(`Copying injection file "${src}" to "${destPath}"`);
     await copyFileOrDir(src, destPath);
   }
 }
 
 function normalizeAppName(appName: string, url: string): string {
-  // use a simple 3 byte random string to prevent collision
-  const hash = crypto.createHash('md5');
-  hash.update(url);
-  const postFixHash = hash.digest('hex').substring(0, 6);
+  // use a simple random string to prevent collision
+  const postFixHash = generatePostFixHash(url);
   const normalized = appName
     .toLowerCase()
     .replace(/[,:.]/g, '')
     .replace(/[\s_]/g, '-');
   return `${normalized}-nativefier-${postFixHash}`;
 }
 
+function generatePostFixHash(seed: string, length = 6): string {
```

Comment:
Three things:

1. Function is a bit weirdly named, it's more `generateRandomPostfix` than a hash.
2. Also, it's absolutely not a big deal here (we don't need strong crypto here), but there's no point using weak `Math.random()`, when we have good RNG APIs in Node. Please replace the Math.random call with a call to https://nodejs.org/docs/latest-v10.x/api/crypto.html#crypto_crypto_randombytes_size_callback or similar
4. Also, since we're doing random anyway, why bother with a seed param? Why not seed with just the call to Crypto.randomSomething ?